labman/db/tests/test_process.py

```diff
@@ -1477,7 +1497,7 @@ def test_format_sample_sheet(self):
         self.assertEqual(exp_sample_sheet, obs_sample_sheet)
 
     def test_generate_sample_sheet(self):
-        # Sequencing run
+        # Amplicon run
         tester = SequencingProcess(1)
```

Comment:
I do not see any tests that AMPLICON sample sheet generation (which uses _generate_amplicon_sample_sheet rather than _format_sample_sheet_data by way of _generate_shotgun_sample_sheet) works correctly in the multi-lane case as well as the single-lane case.  Could we please add one?